grafana-plugin/src/containers/AddResponders/parts/AddRespondersPopup/AddRespondersPopup.tsx

```diff
@@ -97,11 +97,17 @@ const AddRespondersPopup = observer(
     );
 
     const searchForUsers = useCallback(async () => {
-      /**
-       * specifying is_currently_oncall=all will tell the backend not to paginate the results
-       */
-      const userResults = await userStore.search<UserCurrentlyOnCall[]>({ searchTerm, is_currently_oncall: 'all' });
-      setUserSearchResults(userResults);
+      const _searchForUsers = (is_currently_oncall: string) =>
```

Comment:
can't this work with a boolean instead of a string?

Reply:
possibly, I haven't tried. I guess it depends on how our HTTP network package serializes booleans when used as query param value

Reply:
_update_: tested locally, it looks like that works just fine. I've updated this. Thanks for pointing that out